profit_analysis/analysis.py

```diff
@@ -37,6 +38,52 @@
 from mev_inspect.crud.read import read_profit_from_to
 from mev_inspect.web3_provider import W3
 
+USD_PROFIT_FILE_NAME = "usd_profit"
+
+
+def get_max_ind_usd_profit():
+    numbers = []
+    for filename in os.listdir(DATA_PATH):
+        if filename.startswith(USD_PROFIT_FILE_NAME) and filename.endswith(".csv"):
+            match = re.search(r"usd_profit_(\d+)\.csv", filename)
+            if match:
+                number = int(match.group(1))
+                numbers.append(number)
+    if len(numbers) > 0:
+        return max(numbers)
+    else:
+        return -1
+
+
+def read_all_profits(n, save_to_csv=True):
+    total_profit = pd.DataFrame(
+        columns=[
+            BLOCK_KEY,
+            TIMESTAMP_KEY,
+            DATE_KEY,
+            TRANSACTION_HASH_KEY,
+            AMOUNT_RECEIVED_KEY,
+            TOKEN_RECEIVED_KEY,
+            PRICE_RECEIVED_KEY,
+            AMOUNT_DEBT_KEY,
+            TOKEN_DEBT_KEY,
+            PRICE_DEBT_KEY,
+            PROFIT_USD_KEY,
+            CATEGORY_KEY,
+        ]
+    )
+    for i in range(n):
+        profit = pd.read_csv(DATA_PATH + USD_PROFIT_FILE_NAME + f"_{i}.csv")
```

Comment:
I think you can also use the following, which is easier to read:

```suggestion
        profit = pd.read_csv(f"{DATA_PATH}{USD_PROFIT_FILE_NAME}_{i}.csv")
```